posts/Yoann/2017-02-22-hibernate-search-5-7-0-Final-and-5-6-1-Final.adoc

```diff
@@ -0,0 +1,118 @@
+please use Hibernate Search 5.6.1.Final.
+====
+
+== What's new on 5.7 since the candidate release?
+
+Below are the main changes since the candidate release.
+
+For a full list of changes since 5.7.0.CR1, please see https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10061&version=27300[the release notes].
+For a full list of changes since 5.6.0.Final, please see https://hibernate.atlassian.net/issues/?jql=fixVersion%20in%20(5.7.0.Alpha1%2C%205.7.0.Beta1%2C%205.7.0.Beta2%2C%205.7.0.CR1%2C%205.7.0.Final)%20AND%20fixVersion%20not%20in%20(5.6.0.Alpha1%2C%205.6.0.Alpha2%2C%205.6.0.Alpha3%2C%205.6.0.Beta1%2C%205.6.0.Beta2%2C%205.6.0.Beta3%2C%205.6.0.Beta4%2C%205.6.0.CR1%2C%205.6.0.Final)[this list of tickets on our JIRA instance].
+
+=== Common changes
+
+ * https://hibernate.atlassian.net/browse/HSEARCH-2574[HSEARCH-2574]: when it makes sense,
+   you can now repeat Hibernate Search annotations on a single attribute/method.
+   For example, if you want to add two index fields on the same `title` attribute, you can write:
++
+====
+[source, Java]
+----
+    @Field(analyzer = @Analyzer(definition = "myAnalyzer")
+    @Field(name = "title_sort", analyzer = @Analyzer(definition = "myAnalyzerForSort")
+    private String title;
+====
++
+Thus you don't have to use the weird composite annotations (`@Fields`) anymore.
+But of course, those are still supported.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2588[HSEARCH-2588]: some methods in the Sort DSL allowed to specify
+   the sort field type explicitly, which was useful when sorting on fields added through custom field bridges.
+   We improved support for `MetadataProvidingFieldBridge`,
+   so that you now only need to implement this interface on your custom field bridges,
+   and won't need anything special when using the sort DSL anymore.
+   As a result, the `byField(String, SortField.Type)` and `andByField(String, SortField.Type)` methods
+   in the sort DSL have been deprecated.  
+ * https://hibernate.atlassian.net/browse/HSEARCH-2587[HSEARCH-2587]: an NPE could occur when using the Sort DSL to sort on embedded (`@IndexedEmbedded`) fields; this is now fixed.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2576[HSEARCH-2576]: unnecessary memory allocations within queries have
+   been removed, resulting in better query throughput.
+ * And several improvements for other solutions integrating Hibernate Search, like https://hibernate.atlassian.net/browse/HSEARCH-2597[HSEARCH-2597], https://hibernate.atlassian.net/browse/HSEARCH-2561[HSEARCH-2561], https://hibernate.atlassian.net/browse/HSEARCH-2418[HSEARCH-2418] or https://hibernate.atlassian.net/browse/HSEARCH-2585[HSEARCH-2585].
+
+
+=== Elasticsearch-specific changes
+
+ * https://hibernate.atlassian.net/browse/HSEARCH-2453[HSEARCH-2453]: Elasticsearch authentication is now fully supported: there are dedicated `username` and `password` properties so that you don't need to embed the credentials in the URL anymore.
```

Comment:
careful with the usage of "supported" , for some it's a keyword. "implemented" ?